tests/test_milestone7_telemetry.py

```
@@ -0,0 +1,442 @@
+    reg = OrderRegistry(db_file)
+
+    # Count orders before decide
+    orders_before = reg.get_active_orders()
+
+    # Run decide against empty/mocked environment
+    try:
+        decide(target="nonexistent", db_path=db_file)
+    except SystemExit:
+        pass
+    except Exception:
+        pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)